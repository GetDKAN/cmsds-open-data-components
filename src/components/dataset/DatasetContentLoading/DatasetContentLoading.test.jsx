import renderer from 'react-test-renderer'
import DatasetLoading, { DatasetContentLoading, DatasetHeaderLoading } from './DatasetContentLoading'

describe('DatasetContentLoading component.', () => {
  it('DatasetContentLoading matches snapshot.', () => {
    const renderedDatasetContentLoading = renderer.create(
      <DatasetContentLoading />
    ).toJSON()

    expect(renderedDatasetContentLoading).toMatchSnapshot()
  })

  it('DatasetLoading matches snapshot.', () => {
    const renderedDatasetLoading = renderer.create(
      <DatasetLoading />
    ).toJSON()

    expect(renderedDatasetLoading).toMatchSnapshot()
  })

  it('DatasetHeaderLoading matches snapshot.', () => {
    const renderedDatasetHeaderLoading = renderer.create(
      <DatasetHeaderLoading />
    ).toJSON()

    expect(renderedDatasetHeaderLoading).toMatchSnapshot()
  })
})
