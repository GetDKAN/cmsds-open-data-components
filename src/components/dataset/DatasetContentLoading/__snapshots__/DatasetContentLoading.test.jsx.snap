// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DatasetContentLoading component. > DatasetContentLoading matches snapshot. 1`] = `
<div
  className="DatasetContentLoading"
>
  <div
    className="DatasetContentLoadingTable DatasetContentLoading__additional-info"
  >
    <div
      className="DatasetContentLoadingTable__title skeleton-box"
    />
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`DatasetContentLoading component. > DatasetHeaderLoading matches snapshot. 1`] = `
<div
  className="DatasetContentLoading"
>
  <div
    className="DatasetContentLoadingTable DatasetContentLoading__dataset-explorer"
  >
    <div
      className="DatasetContentLoadingTable__title skeleton-box"
    />
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__pagination"
    >
      <div
        className="DatasetContentLoadingTable__pagination_btn skeleton-box"
      />
      <div
        className="DatasetContentLoadingTable__pagination_txt skeleton-box"
      />
      <div
        className="DatasetContentLoadingTable__pagination_btn skeleton-box"
      />
    </div>
  </div>
</div>
`;

exports[`DatasetContentLoading component. > DatasetLoading matches snapshot. 1`] = `
<div
  className="DatasetContentLoading"
>
  <div
    className="DatasetContentLoadingTable DatasetContentLoading__dataset-explorer"
  >
    <div
      className="DatasetContentLoadingTable__title skeleton-box"
    />
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__pagination"
    >
      <div
        className="DatasetContentLoadingTable__pagination_btn skeleton-box"
      />
      <div
        className="DatasetContentLoadingTable__pagination_txt skeleton-box"
      />
      <div
        className="DatasetContentLoadingTable__pagination_btn skeleton-box"
      />
    </div>
  </div>
  <div
    className="DatasetContentLoadingTable DatasetContentLoading__additional-info"
  >
    <div
      className="DatasetContentLoadingTable__title skeleton-box"
    />
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
    <div
      className="DatasetContentLoadingTable__row"
    >
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
      <div
        className="DatasetContentLoadingTable__row_column"
      >
        <div
          className="skeleton-box"
        />
      </div>
    </div>
  </div>
</div>
`;
